nix/haskell-pins.nix

```diff
@@ -214,12 +214,23 @@ let
         sha256 = "sha256-+rHcS+BwEFsXqPAHX/KZDIgv9zfk1dZl0LlZJ57Com4=";
       };
     };
-    # PR: https://github.com/freckle/hspec-junit-formatter/pull/24
-    hspec-junit-formatter = {
-      src = fetchgit {
-        url = "https://github.com/akshaymankar/hspec-junit-formatter";
-        rev = "acec31822cc4f90489d9940bad23b3fd6d1d7c75";
-        sha256 = "sha256-4xGW3KHQKbTL+6+Q/gzfaMBP+J0npUe7tP5ZCQCB5+s=";
+
+    text-icu-translit = {
+      src = pkgs.fetchFromGitHub {
+        owner = "wireapp";
+        repo = "text-icu-translit";
+        rev = "317bbd27ea5ae4e7f93836ee9ca664f9bde7c583";
+        hash = "sha256-9uVqUTkLkE7U19FDjn5xt8JEHyJmosLPSnmW7kYbe5w=";
+      };
+    };
+
+    ghc-source-gen = {
+      version = "0.4.4.0";
+      src = pkgs.fetchFromGitHub {
+        owner = "circuithub";
+        repo = "ghc-source-gen";
+        rev = "7a6aac047b706508e85ba2054b5bedbecfd7eb7a";
+        hash = "sha256-DZu3XAOYLKcSpOYhjpb6IuXMvRHtGohTkL0nsCb/dT0=";
```

Comment:
Please add link to the PR as comments.

Reply:
I don't think we will be able to upstream text-icu-translit, but I will link the one for source-gen.